modules/ROOT/pages/tutorials/nbm-maven-modulesuite.adoc

```diff
@@ -137,217 +134,189 @@ public final class Event {
    public String toString() {
        return index + " - " + date;
    }
-
 }
 ----
 
 
 === Exposing the Package
 
-In this tutorial you will create additional modules that will need to access the  ``Event``  class. In this exercise, you will make the contents of the package that contains the  ``Event``  class public so that other modules can access it. To declare the  ``com.mycompany.mavenselectionsuite``  package as public, you will modify the  ``configuration``  element of  ``nbm-maven-plugin``  in the POM to specify the packages that are exported as public. You can make the changes to the POM in the editor or by selecting the packages to make public in the project's Properties window.
+In this tutorial you will create additional modules that will need to access the  ``Event``  class. In this exercise,
```

Comment:
`=== Exposing the Package`
->

`=== Exporting the Package`

would make it easier to find with search engines